### PR TITLE
[python-frontend] Fix filename for imported modules

### PR DESCRIPTION
**Pull request by brcfarias**

### CODE REVIEW

**Review by lucasccordeiro — CHANGES_REQUESTED** [review]
Can you add the regression test associated with this PR?

File: regression/python/try-fail/main.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from jira import JIRA  # this imported module contains try statements
```
Comment (lucasccordeiro) [inline regression/python/try-fail/main.py:1]: Where is the try-statement in this code? Did I miss something?

**Review by lucasccordeiro — CHANGES_REQUESTED** [review]

File: regression/python/try-fail/main.py (hunk 1 above)
Comment (brcfarias) [reply regression/python/try-fail/main.py:1]: The imported code from this module, which can be located at $HOME/lib/python3.10/site-packages/jira/\_\_init\_\_.py on Linux, contains:

```
"""The root of JIRA package namespace."""
from __future__ import annotations

try:
    import importlib.metadata

    __version__ = importlib.metadata.version("jira")
except Exception:
    __version__ = "unknown"

from jira.client import (
    JIRA,
    Comment,
    Issue,
    Priority,
    Project,
    Role,
    User,
    Watchers,
    Worklog,
)
from jira.config import get_jira
from jira.exceptions import JIRAError

__all__ = (
    "Comment",
    "__version__",
    "Issue",
    "JIRA",
    "JIRAError",
    "Priority",
    "Project",
    "Role",
    "User",
    "Watchers",
    "Worklog",
    "get_jira",
)
```

Comment (brcfarias) [reply regression/python/try-fail/main.py:1]: The issue is that for every import of "module," we generate a corresponding "module.json." In the case of an "import jira," the interpreter resolves this to the file <path>/jira/\_\_init\_\_.py. Thus, we were generating an \_\_init\_\_.json, while the Python converter was expecting a jira.json. This import scenario exercises that condition, and I added this check to demonstrate that the module was imported; however, we do not support the try block contained in the code.

**Review by lucasccordeiro — APPROVED** [review]
Thanks for this clarification, @brcfarias.

**Comment by lucasccordeiro** [comment]
Thanks for submitting this PR, @brcfarias.

**Pull request merged**